cmd/juju/status/status_test.go

```diff
@@ -3838,27 +3962,28 @@ func (s *StatusSuite) TestMigrationInProgress(c *gc.C) {
 
 func (s *StatusSuite) TestMigrationInProgressTabular(c *gc.C) {
 	expected := `
-Model   Controller  Cloud/Region        Version  Notes               SLA
-hosted  kontroll    dummy/dummy-region  1.2.3    migrating: foo bar  unsupported
+Model   Controller  Cloud/Region        Version  SLA          Timestamp       Notes
+hosted  kontroll    dummy/dummy-region  1.2.3    unsupported  15:04:05Z07:00  migrating: foo bar
```

Comment:
should this have the Z in it? I thought it was showing a + or - based on the  TZ offset.

Reply:
This makes me wonder if the first instinct was that it can/should pull the TZ info in the tabular status call. Does the TZ help the user make a right decision in any situation. I supposed if different users of the model are comparing status from different timezones... ok never mind.